Tests/TestPlans/RevenueCat.xctestplan

```diff
@@ -10,7 +10,8 @@
   ],
   "defaultOptions" : {
     "codeCoverage" : false,
-    "testExecutionOrdering" : "random"
+    "testExecutionOrdering" : "random",
+    "testRepetitionMode" : "retryOnFailure"
```

Comment:
Did you mean to leave this too? I use this test plan when I don't wanna run the SK tests

Reply:
Ugh. Sorry. Me rn

![image](https://user-images.githubusercontent.com/1304821/179799420-9d069fbd-37c3-4d8d-9085-c5de9adbf75c.png)

Reply:
Fixed 🤦‍♂️